ui/src/CollectionExerciseDetails.js

```diff
@@ -596,6 +622,15 @@ class CollectionExerciseDetails extends Component {
           <TableCell component="th" scope="row">
             {actionRule.packCode}
           </TableCell>
+          <TableCell component="th" scope="row">
+            <Button
+              variant="contained"
+              onClick={() => this.openCaseCountDialog(actionRule)}
+              id="dryRunCaseCountBtn"
+            >
+              Dry Run
+            </Button>
+          </TableCell>
```

Comment:
Don't know where to write the comment but you should add a `<TableCell></TableCell>` to the `TableHead` so it completes the box. There's no header for the cell at the moment so it looks off